lib/bamboo/mailer.ex

```diff
@@ -183,7 +183,7 @@ defmodule Bamboo.Mailer do
   @doc """
   Deliver an email in the background.
 
-  Same as `deliver_later!/2` but does not return an ok tuple and raises on
+  Same as `deliver_later/2` but does not return an ok tuple and raises on
```

Comment:
I believe this is actually correct as is. 

These docs are for the `deliver_later!/2` function, but here they're saying that the function is the same as `deliver_later/2` (the non-bang version) except `deliver_later!/2` raises on errors. 

Does that make sense? Do you think we could make that more clear?

Reply:
@germsvel That's what I was trying to do, remove the `!`, so it reads `Same as deliver_later/2` now.

Reply:
🤦  I read that backwards, huh? 😅  

Thanks for catching that. Looks good!